spec/services/requirements/access_limit_checker_spec.rb

```diff
@@ -34,5 +34,16 @@
         expect(issues).to be_empty
       end
     end
+
+    context "when user has no organisation associated with account" do
+      let(:edition) { build(:edition, :access_limited, created_by: user) }
```

Comment:
This surprised me a little, as the context and the let are somewhat unrelated. I think there's a way we could tie them together better, but it's just a minor thing.